bin/wabur

```diff
@@ -119,12 +119,14 @@ options = {
 config = WAB::Impl::Configuration.new(usage, options)
 
 if 'new' == config[:mode]
-  dir = config[:base] || '.'
-  dir = File.expand_path(dir)
+  dir = File.expand_path(config[:base] || '.')
   FileUtils.mkdir_p(dir) unless File.expand_path('.') == dir
-  WAB::Impl::Init.setup(File.expand_path(dir), config)
+  WAB::Impl::Init.setup(dir, config)
+  puts "Successfully created a new WAB workspace at #{dir}"
 elsif 'init' == config[:mode]
-  WAB::Impl::Init.setup(File.expand_path('.'), config)
+  dir = File.expand_path('.')
+  WAB::Impl::Init.setup(dir, config)
```

Comment:
Not sure what to say here. A few lines about you combined two lines that did pretty much the same thing and here you separate one line into two. Is there a reason you feel this case is different than the last?

Reply:
on ln#127, the value of `File.expand_path('.')` is cached in local var `dir` which is then used in ln#128 and ln#129